src/report-table/report-table.component.tsx

```diff
@@ -16,14 +16,17 @@ import {
 } from 'carbon-components-react'
 import React, {useEffect, useMemo, useState} from 'react'
 import useSWR, {mutate, SWRResponse} from 'swr'
-import {reportHeaders, defaultPageSize} from '../constants'
+import {reportHeaders, defaultPageSize, documentPath} from '../constants'
 import {ReportTableFetchResponse} from '../types'
 import {fetcher, getReportTableDataURL} from '../utils/lab-orders'
 import classes from './report-table.component.scss'
 import ImagePreviewComponent from '../image-preview-component/image-preview-component'
 
 function getUrl(rows, row) {
-  return rows?.find(intialRow => intialRow.id === row.id)?.url
+  console.log(rows, row)
+  const url = rows?.find(intialRow => intialRow.id === row.id)?.url
+
+  return `${documentPath}${url}`
```

Comment:
`rows` and `row` does not read well... remove console.log... documentPath is used only in this function so no need to manage it into a common location. What about handling null `url` condition?

```
type ReportTableRow = {
  id: string
  tests: string
  url: string
  date: string
  requester: string
  file: string
  conclusion: string
}

function getReportUrl(data: Array<ReportTableRow>, reportId: string) {
  const url = data?.find(e => e.id === reportId)?.url
  return url ? `../document_images/${url}` : ''
}
```
Assuming that if the url is not present then you would return blank...